packages/fast-components-styles-msft/src/utilities/density.ts

```diff
@@ -105,6 +103,41 @@ export function horizontalSpacingNumber(
     );
 }
 
+/**
+ * Returns the width and height for an icon formatted in the provided unit or px by default.
+ *
+ * @param unit The unit of measurement; px by default.
+ */
+export function glyphSize(unit?: string): DesignSystemResolver<string> {
+    return ensureDesignSystemDefaults(
+        (designSystem: DesignSystem): string => {
+            const value: number = glyphSizeNumber()(designSystem);
+            return typeof unit === "string" ? `${value}${unit}` : toPx(value);
+        }
+    );
+}
+
+/**
+ * Returns the width and height for an icon as a number.
+ */
+export function glyphSizeNumber(): DesignSystemResolver<number> {
```

Comment:
I think there would be a better name for this maybe `getGlyphPixelWidthAsNumber`